staging.gen3.biodatacatalyst.nhlbi.nih.gov/manifests/hatchery/hatchery.json

```diff
@@ -171,12 +171,5 @@
       "user-uid": 1000,
       "fs-gid": 100
     }
-  ],
-  "more-configs": [
-    {
-      "type": "dockstore-compose:1.0.0",
-      "path": "/hatchery-more-configs/dicom2.yaml",
-      "name": "Dicom2"
-    }
```

Comment:
I don't know what this is, is it important?

Reply:
Looks like it's from https://github.com/uc-cdis/cdis-manifest/commit/13e0c60921012c327ba17e4d6acc80c5fb988a0f , which @frickjack added 20 days ago. I think that means we might wanna keep it

Reply:
Oh, so he added to PROD but not to PREPROD and the automation erroneously removed it ? 🤕